src/components/dashboard-lists.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
+
+export class Lists extends Component {
+  render() {
+    if (!this.props.username) {
+      return <Redirect to="/" />;
+    }
+    const lists = this.props.lists.map((list, idx) => {
+      console.log(list);
+      return (
+        <li key={idx}>
```

Comment:
I would put a `// FIX` or similar (though react might be spitting at you in warnings) so that when we have identifiers for each list we use them.

Alternatively, set add fake identifiers to your dummy data (in the store). That way, when you start receiving data from the db, you just have to add it to the store in a similar format.